.github/workflows/ci.yml

```diff
@@ -14,12 +14,9 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: ['3.8']
-        toxenv: [django22-celery44-drflatest, django22-celery44-drf312, django22-celery50-drflatest,
-          django22-celery50-drf312, django30-celery44-drflatest, django30-celery44-drf312,
-          django30-celery50-drflatest, django30-celery50-drf312, django31-celery44-drflatest,
-          django31-celery44-drf312, django31-celery50-drflatest, django31-celery50-drf312,
-          django32-celery44-drflatest, django32-celery44-drf312, django32-celery50-drflatest,
-          django32-celery50-drf312, quality, docs]
+        toxenv: [django32-celery44-drflatest, django32-celery44-drf312, django32-celery50-drflatest,
+          django32-celery50-drf312,django40-celery44-drflatest, django40-celery44-drf312,
```

Comment:
we should list these environments in ascending order i.e `drf312` and then `drflatest`